i18n/en/docusaurus-plugin-content-docs/current/user-guide/backend/docker.md

```
@@ -1,56 +1,260 @@
+git clone https://github.com/Open-LLM-VTuber/OLV-Docker-Config
+```
+
+### Download ZIP (for users without coding experience)
+:::warning 
+If you download it this way, the configuration file `conf.yaml` can only be [updated manually](#manual-update).
+:::
+First, [click here to go to the user configuration repository](https://github.com/Open-LLM-Vtuber/OLV-Docker-Config)
+
+Then, on the repository page, click `Code` -> `Download ZIP` to download the Docker user configuration, as shown below.
+
+![](/docker_img/download_docker_conf_repo.png)
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)